src/types/general.ts

```diff
@@ -17,6 +17,11 @@ export enum ItemStatus {
   WontDo = "WontDo",
 }
 
+export enum Role {
+  User = "User",
+  Admin = "Admin"
+}
+
```

Comment:
I'm curious why not place this type in `src/types/user`?

Reply:
Updated! 9250c7e5d3e93f76d8e5ebcbca2acbaa1bba7e77